docs/specs/openapi.yaml

```diff
@@ -70,15 +154,120 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/PostServices"
-              examples:
-                accepted:
-                  value:
-                    type: async
-                    status-code: 202
-                    status: Accepted
-                    change: "1"
-                    result: null
+                $ref: "#/components/schemas/PostServicesResponse"
+              example:
+                {
+                  "type": "async",
+                  "status-code": 202,
+                  "status": "Accepted",
+                  "change": "29",
+                  "result": null
+                }
+  /v1/logs:
+    get:
+      summary: Get service logs
+      tags:
+        - services
+      description: |
+        Fetch previously-written logs from the given services,
+        or request logs from the given services and follows them, in JSON lines format.
+      parameters:
+        - name: services
+          in: query
+          description: |
+            Service name to filter logs by.
+
+            To get logs from multiple services, specify this parameter multiple times.
+
+            If not set, returns logs for all services.
+          schema:
+            type: string
+        - name: follow
+          in: query
+          description: |
+            Whether to follow the logs (keep streaming new logs).
+
+            Must be "true" or "false"; defaults to false.
```

Comment:
The rendered yaml already shows the available values as true and false, and also the default value of false. So I think we can remove this line